grpc/creds/creds_test.go

```diff
@@ -195,8 +196,7 @@ func TestClientReset(t *testing.T) {
 	tc := NewClientCredentials(nil, []tls.Certificate{}, "")
 	_, _, err := tc.ClientHandshake(context.Background(), "T:1010", &brokenConn{})
 	test.AssertError(t, err, "ClientHandshake succeeded with brokenConn")
-	_, ok := err.(interface {
-		Temporary() bool
-	})
+	var hasTemp interface{ Temporary() bool }
+	ok := errors.As(err, &hasTemp)
```

Comment:
I think this can become `errors.Is(err, net.Error)`

Reply:
As discussed offline, errors.As is a type check, errors.Is is a value equivalence check.